DEVELOPING_OPAL/demos/src/main/scala/org/opalj/br/InvokedynamicPrinter.scala

```diff
@@ -32,12 +31,10 @@ object InvokedynamicPrinter extends ProjectAnalysisApplication {
         libcpFiles:              Iterable[File],
         completelyLoadLibraries: Boolean,
         fallbackConfiguration:   Config
-    )(
-        implicit
-        initialLogContext: LogContext
-    ): Project[URL] = {
+      )(implicit
```

Comment:
I guess this is the best we can achieve for implicit parameter lists, which would be setting
`newlines.implicitParamListModifierPrefer = after`
The old behavior would require setting that `Prefer` setting to both, before and after, but it only takes one of them, while there is also a `Force` setting that allows setting both but would always do this, even if it fits into a single line.

Cf. https://www.github.com/scalameta/scalafmt/issues/2279